platform-sdk/swirlds-common/src/main/java/com/swirlds/common/stream/internal/LinkedObjectStreamValidateUtils.java

```diff
@@ -239,7 +239,7 @@ public static boolean verifySignature(final Hash hash, final Signature signature
                     SignatureType.RSA.signingAlgorithm(), SignatureType.RSA.provider());
             sig.initVerify(publicKey);
             hash.getBytes().updateSignature(sig);
-            return sig.verify(signature.getSignatureBytes());
+            return sig.verify(signature.getBytes().toByteArray());
```

Comment:
here you can use `signature.getBytes().verifySignature(sig)` to avoid copying the byte array

Reply:
done